### PR TITLE
#44 penalty trigger

### PR DESCRIPTION
**Pull request by bubriks**
please check it out and tell me if this is what you needed me to do in this branch

### CODE REVIEW

File: Code/Database/SQLCreateQuery.sql [hunk 1]
```diff
@@ -89,14 +89,16 @@ CREATE TABLE Copy (
 CREATE TABLE Borrow (
```
Comment (RaidenRabit) [inline Code/Database/SQLCreateQuery.sql]: Loan

File: Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs [hunk 2]
```diff
@@ -50,5 +51,45 @@ public bool ReturnBook(int copyId)
...
+                if (DateTime.Now >= borrow.FromDate.AddDays(member.MemberType.LendingLenght + member.MemberType.GracePeriod) && borrow.noticeSent == null)
+                {
+                    borrow.noticeSent = false;
+                }
+            }
+            return _lendingDa.SaveBorrowChanges();
+        }
+    }
+
+    public class NoticeTimer
```
Comment (RaidenRabit) [inline Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs]: I know its just a helper class, but i think it should be in model layer, not in dm

**Review by RaidenRabit — CHANGES_REQUESTED** [review]

File: Code/Database/SQLCreateQuery.sql (hunk 1 above)
Comment (bubriks) [reply Code/Database/SQLCreateQuery.sql]: this ill fix (idk if in this pr is good though- maybe new pr would be better or something idk)

File: Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs (hunk 2 above)
Comment (bubriks) [reply Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs]: but model layer cant call service

Comment (RaidenRabit) [reply Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs]: then perhaps you should add a timerDm as well

Comment (bubriks) [reply Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs]: ok that i can do

Comment (bubriks) [reply Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs]: enything else i should fix/do in this pr?

**Review by RaidenRabit — APPROVED** [review]

**Pull request merged**